src/libraries/Microsoft.PowerFx.Core/Public/DisplayNameCollisionException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx.Core
+{
+    public class DisplayNameCollisionException : Exception
```

Comment:
>DisplayNameCollisionException [](http://example.com/codeflow?start=17&length=29)

Do we need a dedicated exception here? Or just a generic Argument exception?

Reply:
I prefer a dedicated expression here, but I'm happy to hear why we shouldn't do that.